packages/deployer/ens.js

```diff
@@ -118,6 +185,35 @@ class ENS {
     }
   }
 
+  //this method assumes that the current owner is `from`,
+  //and sets the new owner to be `owner`.
+  async alienateNameOwner({ name, from, owner }) {
```

Comment:
would something like `changeNameOwner` be a more appropriate name for this method?

Reply:
Well, the point is that this is for giving it away, given that you currently own it.  I'm not sure that it would work otherwise.  So it could be called `giveAwayNameOwner` or something, but I wouldn't want to just use the word "change" or "update"; that doesn't seem specific enough.

Reply:
Isn't this method really just something that takes the current owner and a new owner and then changes ownership?

Reply:
No, it assumes you already own it.  Otherwise you're not going to be able to give it away!

That said, if I redo this the way you suggested below, this method is likely just going to go away, replaced by more direct calls.

Reply:
OK, I ended up changing this to a more general update method since there's no real need to do the tree stuff.  Yay simplicity!